test/models/host_status/configuration_status_test.rb

```diff
@@ -99,10 +99,11 @@ def setup
       end
 
       def stub_outofsync_setting(value)
-        Setting.create(name: :testorigin_out_of_sync_disabled,
-                       description: 'description', default: false)
-        Foreman.settings._add('testorigin_out_of_sync_disabled', category: 'Setting::General', description: 'description', default: false)
-        Foreman.settings.load
+        Foreman.settings._add('testorigin_out_of_sync_disabled', context: :test,
```

Comment:
Nit: Perhaps this is easier to read (if also applied to others)?
```suggestion
        Foreman.settings._add('testorigin_out_of_sync_disabled',
                                              context: :test,
```

Reply:
This RuboCop would not agree to, it would need to be:
```suggestion
        Foreman.settings._add('testorigin_out_of_sync_disabled',
          context: :test,
```
maybe
```suggestion
        Foreman.settings._add('testorigin_out_of_sync_disabled',
                                              context: :test,
```

Reply:
Fixed that :)